tests/test_potential.py

```diff
@@ -3645,6 +3646,62 @@ def test_RotateAndTiltWrapper():
     NFW_wrapped= potential.RotateAndTiltWrapperPotential(zvec=zvec, galaxy_pa=galaxy_pa, offset=[20.,0.,3.], pot=potential.TriaxialNFWPotential(amp=1.,b=0.7,c=0.5))
     NFW_rot= potential.TriaxialNFWPotential(amp=1., zvec=zvec, pa=galaxy_pa,b=0.7,c=0.5)
     assert (evaluatePotentials(NFW_wrapped, 0., 0., phi=0.)-evaluatePotentials(NFW_rot, 20., - 3., phi=numpy.pi)) < 1e-6, 'Wrapped + Offset and Internally rotated NFW potentials do not match when evaluated at the same point'
+    #test a quick orbit integration to hit the C code (also test pure python)
```

Comment:
Let’s  split this off into a separate test

Reply:
ok, done - the problem seems to be that the offset is not being applied at all in the C code... I can't quite see why though? If I integrate the same orbit in a regular and offset potential, I get the same result when using the C integrators, but not when using the pure python (note that the test passes for pure python).

I initially thought this was a problem to do with the caching, but after doing some tests I can't see that that is the issue. It's probably blindingly obvious 😬

Reply:
I think the currently implemented C code is doing things a little wrongly, since it applies the offset _only_ to the cached values. Just pushing up my current version, which fixes that...

Reply:
here's a quick gist showing the problem (still persisting in the code I just uploaded): https://gist.github.com/jmackereth/94dd533eb91c1ddc24931489573a2d69

Reply:
Thanks for the notebook! I just ran this and it works for me, tests also pass... Are you sure you re-compiled?